dangerfile.js

```diff
@@ -67,17 +69,25 @@ const addClickupRefComment = async (taskId, isCustom) => {
 
 const parallelRequests = (tasks = [], req) => {
   if (tasks.length === 0) {
-    return;
+    return[];
   }
```

Comment:
nit: I believe this check can be removed. If `tasks` is empty the `Promise.all(...)` should return an empty array anyway.